iothub/device/src/Serialization/Utf8PayloadEncoder.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// A UTF-8 PayloadEncoder implementation.
+    /// </summary>
+    public class Utf8PayloadEncoder : PayloadEncoder
+    {
+        private Utf8PayloadEncoder()
```

Comment:
private? Why not internal or even protected internal?

Reply:
sure, that makes sense

Reply:
Looks like we're exposing a public static instance of this class which doesn't require any additional customization before being used. That should be enough for both users wanting to use this class or test something that uses this class. So, I'm inclined to not add additional visibility on this private constructor. What do you think?